pr_check.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Just need to replace this file with one like this: https://github.com/RedHatInsights/sources-monitor-go/blob/main/pr_check.sh

We don't want to deploy the eph env for this or anything yet just because we don't have proper smoke tests quite yet.